index.js

```diff
@@ -74,6 +75,27 @@ async function startProcessEngine(sqlitePath) {
     await bootstrapper.start();
 
     logger.info('Bootstrapper started successfully.');
+
+    const httpExtension = await container.resolveAsync('HttpExtension');
+    httpExtension.app.get('/', (request, response) => {
+      const name = packageJson.name;
+      const version = packageJson.version;
+      const license = packageJson.license;
+      const contributors = packageJson.contributors;
+
+      const returnObject = {
+        name: name,
+        version: version,
+        license: license,
+        contributors: contributors,
+      };
+
+      response
+        .status(200)
+        .header('Content-Type', 'application/json')
+        .send(JSON.stringify(returnObject, null, 2));
```

Comment:
That's whats `response.json()` is for 😉 

```suggestion
response
  .status(200)
  .json(returnObject);
```